install/db/achecker_schema.sql

```diff
@@ -308,7 +308,7 @@ CREATE TABLE `themes` (
   `title` varchar(80) NOT NULL DEFAULT '',
   `version` varchar(10) NOT NULL DEFAULT '',
   `dir_name` varchar(20) NOT NULL DEFAULT '',
-  `last_updated` date NOT NULL DEFAULT '0000-00-00',
+  `last_updated` date NOT NULL DEFAULT '2000-01-01',
```

Comment:
Instead of setting to an ambiguous initial date, using NULL as the default might be a better option.

Reply:
Thanks, will do that